From 166cfd312be3f47cfca9b66a20f0fca9bdf8ba3b Mon Sep 17 00:00:00 2001
From: "Leo (Sunpeng) Li" <sunpeng.li@amd.com>
Date: Mon, 20 Nov 2017 13:16:06 -0500
Subject: [PATCH 4427/5855] drm/amd/display: Disable freesync by default

Freesync properties are being set to the 'freesync_capable' flag,
which will enable freesync on all freesync capable displays. Don't do
this, since freesync should be disabled by default regardless of
capabilities.

Signed-off-by: Leo (Sunpeng) Li <sunpeng.li@amd.com>
Reviewed-by: Harry Wentland <Harry.Wentland@amd.com>
---
 drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c b/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c
index 7bf66c9..fe3659a 100644
--- a/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c
+++ b/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c
@@ -5063,10 +5063,6 @@ void amdgpu_dm_add_sink_to_freesync_module(
 			dm_con_state->freesync_capable = true;
 		}
 	}
-
-	dm_con_state->user_enable.enable_for_gaming = dm_con_state->freesync_capable;
-	dm_con_state->user_enable.enable_for_static = dm_con_state->freesync_capable;
-	dm_con_state->user_enable.enable_for_video = dm_con_state->freesync_capable;
 }
 
 void amdgpu_dm_remove_sink_from_freesync_module(
-- 
2.7.4

